content/news/XXXX-XX-XX-mixxx-2-3-0-released.md

```diff
@@ -0,0 +1,154 @@
+
+For the full list of changes, have a look at the [changelog](https://manual.mixxx.org/2.3/en/chapters/appendix/version_history.html) and the [2.3.0 milestone on Launchpad](https://launchpad.net/mixxx/+milestone/2.3.0).
+
+**Note:** KDE users may see broken icons due to [a bug in the kIconThemes 5.80 package](https://bugs.kde.org/show_bug.cgi?id=434451). If you're affected, please uninstall the package or update it to version 5.82. Unfortunately, not all Linux distributions have updated the package yet. For Ubuntu 21.04, a patched version of the package has been added to the [Mixxx PPA](https://launchpad.net/~mixxx/+archive/ubuntu/mixxx/+packages).
+
+This has been a rather big release, with lots of useful changes, bug fixes and improvements.
+All in all, with a total of 7477 changes over 1 million lines of code were modified since the [2.2.4 release]({filename}/news/2020-06-11-Mixxx-2-2-4-released.md)!
+
+<!-- Numbers acquired using `git shortlog -sn release-2.2.4..upstream/2.3 | wc -l`, `git log --oneline --no-merges release-2.2.4..upstream/2.3 -- . ':!lib' | wc -l` and `git diff --shortstat release-2.2.4..upstream/2.3 -- . ':!lib`. -->
+
+Despite our plans to release Mixxx more often, it's been two years since the 2.2.x release, and 2.3 has been in the beta phase for almost a year now.
+The reason for these delays is that we [switched to the CMake build system generator]({filename}/news/2020-12-14-scons-cmake-migration.md), worked on some big refactorings and [infrastructure updates]({filename}/news/2021-02-23-build-infrastructure-updates.md) that were necessary to streamline our process, but also took a lot of time.
+
+Our small development team can always need a helping hand, so if you want to help out and make Mixxx better, [get in touch with us]({filename}/pages/get-involved.md)!
+Unfortunately, we receive almost no C++ contributions from Windows or macOS developers despite our large user base.
+Our whole core development team uses Linux, which makes it extremely hard to maintain support for these systems.
+Without significantly more contributions, the future of Mixxx on Windows and macOS is at stake.
+If you know a bit of C++ and use Windows or macOS, please consider helping out.
```

Comment:
@mixxxdj/developers Please check if this is okay or if something should be changed.